cmake/cmake_uninstall.cmake

```diff
@@ -0,0 +1,32 @@
+# http://www.vtk.org/Wiki/CMake_FAQ#Can_I_do_.22make_uninstall.22_with_CMake.3F
+
+IF(NOT EXISTS "/home/san/somecode/satellogic/gr-dvb/cmake/install_manifest.txt")
```

Comment:
Likewise, please make this generic without hard coding the path "/home/san/somecode/satellogic/gr-dvb"